Services.Test/Concurrency/PerMinuteCounterTest.cs

```diff
@@ -0,0 +1,76 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System;
+using Microsoft.Azure.IoTSolutions.DeviceSimulation.Services.Concurrency;
+using Services.Test.helpers;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Services.Test.Concurrency
+{
+    public class PerMinuteCounterTest
+    {
+        private static ITestOutputHelper log;
+
+        private readonly TargetLogger targetLogger;
+
+        public PerMinuteCounterTest(ITestOutputHelper logger)
+        {
+            log = logger;
+            this.targetLogger = new TargetLogger(logger);
+        }
+
+        /**
+         * Calls are slowed down only *after* reaching the limit for events
+         * per minute. So, when the limit is 60 events/minute, 60 events should
```

Comment:
do limitations/minute make sense to test?  per second and per day are the hub throttles.